server/routes/router.js

```diff
@@ -4,24 +4,20 @@ var axios = require('axios');
 var users = require("../models/userAuth");
 var mongoose = require('mongoose');
 
-
 //index router
-router.get('/' , (req,res) => {
+router.get('/', (req, res) => {
```

Comment:
I think it might be a good idea to choose between single and double quotes amd keep it in the file / codebase.

Reply:
Agreed